sqlserver/CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
+
+* [Added] Add improved database file IO metrics and tags. See [#10901](https://github.com/DataDog/integrations-core/pull/10901).
+* [Added] sqlserver: Add statement metadata to events and metrics payload. See [#10881](https://github.com/DataDog/integrations-core/pull/10881).
+* [Added] Add option to disable query metrics secondary aggregates for user and database. See [#10975](https://github.com/DataDog/integrations-core/pull/10975).
+* [Added] Add plan handle to the plan event . See [#10939](https://github.com/DataDog/integrations-core/pull/10939).
+* [Added] Add the option to set a reported hostname (SQLServer). See [#10688](https://github.com/DataDog/integrations-core/pull/10688).
+* [Added] Add `enforce_collection_interval_deadline` option to set plan collection deadline. See [#10829](https://github.com/DataDog/integrations-core/pull/10829).
+* [Added] Add current timestamp to SQLServer activity collection. See [#10786](https://github.com/DataDog/integrations-core/pull/10786).
+* [Added] Update SQLServer activity query to get timestamps with the timezone. See [#10782](https://github.com/DataDog/integrations-core/pull/10782).
+* [Added] Add active sessions monitoring support for SQLServer. See [#10610](https://github.com/DataDog/integrations-core/pull/10610).
+* [Fixed] Bump base package. See [#11067](https://github.com/DataDog/integrations-core/pull/11067).
+* [Fixed] add comment to autogenerated model files. See [#10945](https://github.com/DataDog/integrations-core/pull/10945).
+* [Fixed] fix dangling connection on unhandled exception bug. See [#10935](https://github.com/DataDog/integrations-core/pull/10935).
+* [Fixed] Improve plan lookup performance. See [#10828](https://github.com/DataDog/integrations-core/pull/10828).
+* [Fixed] fix missing ConnectionRetryCount when DSN set. See [#10830](https://github.com/DataDog/integrations-core/pull/10830).
+* [Fixed] Fix wrong example for MultiSubnetFailover in connection_string example. See [#10832](https://github.com/DataDog/integrations-core/pull/10832).
+* [Fixed] improve sqlserver agent query performance for DBM metrics query. See [#10810](https://github.com/DataDog/integrations-core/pull/10810).
+* [Fixed] Properly filter databases when using database autodiscovery. See [#10416](https://github.com/DataDog/integrations-core/pull/10416).
+* [Fixed] Update SQLServer Query Metrics Collection Query to Improve Performance. See [#10763](https://github.com/DataDog/integrations-core/pull/10763).
+* [Fixed] Add ConnectRetryCount to connection string. See [#10738](https://github.com/DataDog/integrations-core/pull/10738).
+* [Fixed] fix broken unicode support for sqlserver. See [#10713](https://github.com/DataDog/integrations-core/pull/10713).
+* [Fixed] enable autocommit for pyodbc in sqlserver. See [#10717](https://github.com/DataDog/integrations-core/pull/10717).
+* [Fixed] fix broken unicode support for sqlserver. See [#10713](https://github.com/DataDog/integrations-core/pull/10713).
```

Comment:
Lines 26 and 27 are duplicates of 24 and 25.

Reply:
Good catch!